.github/workflows/nightly.yml

```diff
@@ -62,7 +62,7 @@ jobs:
 
       # Run Selenium tests on XSK
       - name: Run Integration & Selenium Tests
-        run: mvn clean install -P itests -Dhana.url="${{secrets.ITESTS_HANA_URL}}" -Dhana.username="${{secrets.ITESTS_HANA_USERNAME}}" -Dhana.password="${{secrets.ITESTS_HANA_PASSWORD}}"
+        run: mvn clean install -P itests -fn -Dhana.url="${{secrets.ITESTS_HANA_URL}}" -Dhana.username="${{secrets.ITESTS_HANA_USERNAME}}" -Dhana.password="${{secrets.ITESTS_HANA_PASSWORD}}"
```

Comment:
Does this fail the build at the end if there are test failures? I read here: https://maven.apache.org/ref/3.6.3/maven-embedder/cli.html that `-fn` never fails the build but I haven't tried it. Did you check it?

Reply:
You were correct, after testing it `-fn` didn't fail the builds, however there is an alternative.
If you build only the `integration-tests` module with the `-fae` flag it runs all the tests regardless of the previous failures and fails at the end of the build. This didn't work if i start a build from the root directory, so i am building only the integration-tests module. This requires a previous build of xsk on the container, so that the classes used in the HDB integration tests are available, but this is not an issue for the nightly or a future move of this workflow in a PR build (where XSK would already have to be built).

Reply:
I've already approved but could you try instead of `mvn clean install -P itests` to run `mvn test`? I think we could only run the `test` goal and also do we need the `itests` profile if we run mvn inside the integration tests directory?